public/javascript/pump/view.js

```diff
@@ -630,6 +631,11 @@
                 streams.notifications = view.minorStreamView.model;
             }
             return streams;
+        },
+        closeDropdown: function(e) {
+            // Close by trigger because call function 'dropdown'
+            // in element parent or child, block navigation
+            $("#fat-menu.dropdown.open").trigger("click");
```

Comment:
It's different, `.hide()` add a class to hide the element, but you need change all dropdown behaviour in other parts of the code and need call manually `.show()`, `.trigger()` only call a action similar to user interaction and dropdown works with normal behaviour.

if you call `$("#fat-menu.dropdown.open .dropdown-menu").hide()` and click in menu again never show again the menu